packages/core/src/Stores/client-store.js

```diff
@@ -1021,8 +1025,9 @@ export default class ClientStore extends BaseStore {
 
         let is_allowed_real = true;
         // Performs check to avoid login of landing companies that are currently not supported in app
+        // TODO: [deriv-eu] Remove this after full merging of EU. When EU is enabled all landing companies are allowed.
         account_list.forEach(function(account) {
-            if (!/^virtual|svg$/.test(account.landing_company_name)) {
+            if (!this.is_eu_enabled && !/^virtual|svg$/.test(account.landing_company_name)) {
```

Comment:
Then this check won't be required here.

Reply:
@reza-fs I applied your prev comment but I don't get this one! if I remove this check even when the EU is NOT enabled, `is_allowed_real` is always true. is that what we want?